alerting/src/main/kotlin/org/opensearch/alerting/DocumentLevelMonitorRunner.kt

```diff
@@ -321,7 +308,15 @@ class DocumentLevelMonitorRunner : MonitorRunner() {
                                                         ) {
                                                         override fun handleException(e: TransportException) {
                                                             logger.error("Fan out retry failed in node ${localNode.id}")
```

Comment:
plz log exception. If we dont log exception debugging is tough .

Reply:
addressed this.